_extras/design.md

```diff
@@ -118,10 +118,84 @@ How do I...
 ### Variables and Assignment (9:15)
 
 *   Teaching: 10 min
+
+Basic assignment of a value.  What kind of thing is it?  Can assign results
+of computation and use a variable on the right to assign to the left.
+
+```
+a = 3
+print(a)
+type(a)
```

Comment:
Methodological question, then.  This was presuming learners typing along at some sort of interactive python prompt, not running these snippets as a block.  If we use `print(type(a))` how much explanation of the `print()` black box do we explain, and where?  We're already at function composition.  :-)